cli/README.md

```diff
@@ -0,0 +1,46 @@
+# AMP-Toolbox CLI
+
+A command line interface (CLI) for AMP-Toolbox
+
+Supported commands:
+- [update-cache](#update-cache): CLI for the [AMP update-cache](https://developers.google.com/amp/cache/update-cache).
+- [version](#version): Prints the current version.
+- [help](#help): Displays available commands.
+
+## Commands
+
+### help
+
+Displays the help menu, listing available commands:
+```
+./amp-toolbox-cli help
+```
+
+It also supports help for one of the available commands:
```

Comment:
suggestion: 'Pass a command to get more information about this specific command.`